docs/oauth-cli.md

```diff
@@ -0,0 +1,109 @@
+# Testkube CLI Authentication
+
+Testkube doesn't provide a separate user/role management system to protect access to its CLI.
+Users can configure OAuth based authenication module using Testkube Helm chart parameters and
+CLI config command.
+Testkube can automatically configure Kubernetes Nginx Ingress Controller and create required 
```

Comment:
Testkube can automatically configure the Kubernetes Nginx Ingress Controller

Reply:
fixed